packages/local-cli/generator/templates.js

```diff
@@ -25,21 +25,20 @@ function createProjectFromTemplate(
   template,
   yarnVersion
 ) {
-  // Expand the basic 'HelloWorld' template
-  copyProjectTemplateAndReplace(
-    path.resolve(__dirname, '../templates/HelloWorld'),
-    destPath,
-    newProjectName
+  const templatePath = path.resolve(
+    process.cwd(),
```

Comment:
I think this needs to be `destPath` (the project directory, passed from `init()`) instead of the current working directory (which could be any dir where the command to create a project was run).

Reply:
actually we could just use `require.resolve('react-native/template')` instead, since we are in context of `node_modules`, right?

Note that `destPath` is still passed on line 32. The diff might be confusing, but we are only changing the "path.resolve(__dirname, '../templates/HelloWorld')," part.

Reply:
Yeah, `require.resolve` should work I think.